README.md

```diff
@@ -46,160 +45,38 @@ Lynx is a Fast, Secure and Reliable Terraform Backend. It is built in Elixir wit
 
 - **OAuth2 Authentication Support:** Support for OAuth2 Providers like Azure AD OAuth, Keycloak, Okta ... etc
 
+## Deployment
 
-### Docker Deployment
-
-1. To run with docker and docker-compose. Make sure you update `server_name` in `nginx.conf` file.
-
-```zsh
-$ git clone https://github.com/Clivern/Lynx.git app
-$ cd app
-$ docker-compose up -d
-```
-
-Here is a [video demonstration](https://www.youtube.com/watch?v=YNkHfysr3-0)
-
-
-### Ubuntu Deployment
-
-1. Install elixir and `PostgreSQL`
-
-```zsh
-$ apt-get update
-$ apt-get upgrade -y
-$ apt-get install -y postgresql \
-    elixir \
-    erlang-dev \
-    make \
-    build-essential \
-    erlang-os-mon \
-    inotify-tools \
-    erlang-xmerl
-```
-
-2. Setup `PostgreSQL` database, username and password
-
-```zsh
-# Create PostgreSQL user with password
-$ sudo -u postgres psql -c "CREATE USER lynx WITH PASSWORD 'lynx';"
-$ sudo -u postgres psql -c "ALTER USER lynx CREATEDB;"
-
-# Create database
-$ sudo -u postgres psql -c "CREATE DATABASE lynx_dev OWNER lynx;"
-```
-
-3. Configure Environment Variables: Set up the required environment variables from `.env.example`.
-
-```zsh
-$ mkdir -p /etc/lynx
-$ cd /etc/lynx
-$ git clone https://github.com/Clivern/Lynx.git app
-$ cd /etc/lynx/app
-$ cp .env.example .env.local # Adjust the database configs
-```
-
-4. Migrate the database
-
-```zsh
-$ make deps
-$ make migrate
-```
-
-5. Create a systemd service file `/etc/systemd/system/lynx.service`
-
-```zsh
-[Unit]
-Description=Lynx
-
-[Service]
-Type=simple
-Environment=HOME=/root
-EnvironmentFile=/etc/lynx/app/.env.local
-WorkingDirectory=/etc/lynx/app
-ExecStart=/usr/bin/mix phx.server
-
-[Install]
-WantedBy=multi-user.target
-```
-
-```zsh
-$ systemctl enable lynx.service
-$ systemctl start lynx.service
-```
-
-
-### Development
-
-To run `postgresql` with `docker` or `podman`
-
-```zsh
-$ docker run -itd \
-    -e POSTGRES_USER=lynx \
-    -e POSTGRES_PASSWORD=lynx \
-    -e POSTGRES_DB=lynx_dev \
-    -p 5432:5432 \
-    --name lyx \
-    postgres:15.2
-
-$ podman run -itd \
-    -e POSTGRES_USER=lynx \
-    -e POSTGRES_PASSWORD=lynx \
-    -e POSTGRES_DB=lynx_dev \
-    -p 5432:5432 \
-    --name lyx \
-    postgres:15.2
+Lynx requires a modern postgres version to function. No Object Storage required
```

Comment:
can this line be like the following since ecto support most postgresql versions https://github.com/elixir-ecto/postgrex

```
Lynx requires a [PostgreSQL](https://www.postgresql.org/) database. No Object Storage is required
```